### PR TITLE
transmogrification from old to the new hotness IE moving from .gitlab…

### PR DESCRIPTION
**Pull request by notjames**
…-ci to jenkins :(

### CODE REVIEW

File: charts/curator/values.yaml [hunk 1]
```diff
@@ -4,9 +4,7 @@
 # name: value
 
 name: curator
-image:
-   name: quay.io/samsung_cnct/curator-container
-   tag: latest
+image: quay.io/samsung_cnct/curator-container:latest
```
Comment (coffeepac) [inline charts/curator/values.yaml]: can we default to a specific version?

File: pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+type: chart
+rootfs:
+  - image: samsung_cnct/chart-curator
```
Comment (davidewatson) [inline pipeline.yaml]: I thought this was for the container image?

File: charts/curator/Chart.yaml [hunk 3]
```diff
@@ -1,5 +1,6 @@
+---
 name: curator
-version: ${CHART_VER}-${CHART_REL}
+version: 0.0.1
```
[the remark this answers is not in the document]
Comment (maratoid) [reply charts/curator/Chart.yaml]: It does not need to match the versionfile. Versionfile overrides the version numbers in charts.

[the remark this answers is not in the document]
Comment (davidewatson) [reply charts/curator/Chart.yaml]: Only `.versionfile` matters. This is just filler.

BTW, `0.1.0` is typically the [first revision](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase) but it matter not.

[the remark this answers is not in the document]
Comment (notjames) [reply charts/curator/Chart.yaml]: Thanks @davidewatson  :) Good heads up.

**Pull request merged**